pkg/kubectl/history.go

```diff
@@ -55,11 +54,11 @@ type HistoryViewer interface {
 
 func HistoryViewerFor(kind schema.GroupKind, c clientset.Interface) (HistoryViewer, error) {
 	switch kind {
-	case extensions.Kind("Deployment"), apps.Kind("Deployment"):
+	case extensionsv1beta1.Kind("Deployment"), apps.Kind("Deployment"):
```

Comment:
This is cool, but would you use this typevisitor library instead?  That way we don't need to update this code each time that we add a new version or group for the type.

https://github.com/kubernetes/kubernetes/blob/e8e617f31dfbfe2ad8ad4743cfcc17ebd6f283a1/pkg/kubectl/apps/kind_visitor.go#L27

Reply:
@pwittrock I have tried studying the KindVisitor interface, but I still do not understand how to use it here. Please I would like to get more clarification.

Reply:
Don't worry about it for now.  Can get this later.